client/my-sites/plugins/plugins-list/style.scss

```diff
@@ -1,5 +1,53 @@
 .plugins-list {
 	margin-bottom: 16px;
+
+	.plugin-item {
```

Comment:
I'm not 100% sure if I understand the guideline correctly but, I always avoid nesting like this. (See the "Nesting" section) https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md?term=css

This looks like an emergency fix, so this alone should not stop this PR to be merged, but I also want to know what is the "right" way that follows the guideline :)

Reply:
Hello @iamtakashi, do you mean the recommendation to not use SCSS nesting and rather write the rule like classic CSS?
```css
.plugins-list .plugin-item {}
```
That's a good point, I'll fix that before merging.

Or do you thing there should be no nesting of `.plugin-item` inside `.plugins-list` at all? Here I'm not so sure -- nesting is a good way how to lay out items inside a container:
```css
.horizontal-toolbar .button {
  margin-left: 6px;
}
.vertical-list .button {
  margin-top: 6px;
}
```
Where else would I specify the container-specific margins? The nesting can be very dangerous though -- if we nested a `.horizontal-toolbar` inside a `.vertical-list`, then there are two conflicting `.button` styles. That's why I think it's recommended to avoid nesting if possible. But I don't know a better solution for this case.

Reply:
@jsnajdr Yes, I only meant this. Thanks for confirming and fixing this before merging.

> do you mean the recommendation to not use SCSS nesting and rather write the rule like classic CSS?